keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ApplicationEvent.kt

```diff
@@ -32,7 +32,9 @@ abstract class ApplicationEvent(
 data class ApplicationActuationPaused(
   override val application: String,
   override val timestamp: Instant,
-  override val triggeredBy: String?
+  override val triggeredBy: String?,
+  override val level: EventLevel = EventLevel.WARNING,
```

Comment:
nit: can you import these all so that they just show up as `WARNING`? In intellij you can option + click to show context actions, and it is something like "add static import"

Reply:
(for all the enum values)

Reply:
Not sure that I understand :)

Reply:
`import com.netflix.spinnaker.keel.events.EventLevel.*;` or one by one?

Reply:
one by one